Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift

```diff
@@ -146,18 +153,48 @@ class ClaimSafeTokenFlow: UIFlow {
         show(vc, crossDissolve: true)
     }
 
-    func chooseTutorial() {
-        let vc = factory.chooseTutorial { [unowned self] in
-            tokenDistribution()
+    func showDelegateIntro() {
+        let vc = factory.chooseDelegateIntro { [unowned self] in
+            showGuardian()
+        } onCustomAddress: { [unowned self] in
+            enterCustomAddress()
         }
         show(vc)
     }
 
-    func tokenDistribution() {
-        let vc = factory.tokenDistribution { [unowned self] in
-            showDisclaimer()
+    func showWhatIsSafe() {
+        let vc = factory.chooseWhatIsSafe { [unowned self] in
+            showTokenDistribution()
         }
+        show(vc)
+    }
 
+    func showGuardian() {
```

Comment:
this is wrong, i.e. this code was deleted before and it should use the "SelectDelegate" flow